packages/hoppscotch-common/src/components/smart/EnvInput.vue

```diff
@@ -176,10 +178,11 @@ const handleKeystroke = (ev: KeyboardEvent) => {
     return
   }
 
-  showSuggestionPopover.value = true
+  // show suggestions only when the user types a character other than enter or tab
+  showSuggestionPopover.value = !["Enter", "Tab"].includes(ev.key)
 
   if (
-    ["Enter", "Tab"].includes(ev.key) &&
+    ["Enter"].includes(ev.key) &&
```

Comment:
Do we need the above condition since we intend not to show suggestions while pressing `Enter`?